tests/unit/forklift/test_legacy.py

```diff
@@ -919,6 +919,7 @@ def test_fails_with_invalid_names(self, pyramid_config, db_request, name):
             "for more information."
         ).format(name)
 
+    @pytest.mark.xfail(reason='https://github.com/pypa/warehouse/issues/4079')
```

Comment:
`black` will want these to be `"` instead of `'`, though that landed while I was away :-D not sure if we enforce it in linter yet.